src/DotNetEnv/DotNetEnv.csproj

```diff
@@ -4,6 +4,7 @@
     <Description>A .NET library to load environment variables from .env files</Description>
     <AssemblyTitle>DotNetEnv</AssemblyTitle>
     <PackageVersion>1.1.0</PackageVersion>
+    <LangVersion>7.1</LangVersion>
```

Comment:
Do we really need to set the LangVersion for these changes?  I am not sure how much of an impact this has, but I don't want to restrict anyone from attempting to use older code with this library

Reply:
AFAIK it's only the compiler that reads the property to see what language features it should support, but it's not strictly necessary, so it's not a problem to remove it.